Ch setup codeclimate 163496103

kenzdozz opened the pull request:
#### What does this PR do?
Integrated codeclimate maintainability
#### Description of Task to be completed?
Added triggers on travis.yml to run coverage 
Linked repository to codeclimate
Added codeclimate maintainability badge to readme
#### How should this be manually tested?
Check the readme for maintainability badge from codeclimate
#### What are the relevant pivotal tracker stories?
#163496103

ascii-dev commented:
Hi @kenzdozz, any reason why you have 10 commits for this task. I think you should combine them into one commit. Thanks

ascii-dev commented on .travis.yml:
```diff
@@ -1,4 +1,17 @@
+# before_script:
+#   - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+#   - chmod +x ./cc-test-reporter
+#   - ./cc-test-reporter before-build
```
Any reason for pushing comments? Thanks

ascii-dev commented on .travis.yml:
```diff
@@ -1,4 +1,17 @@
+
+after_script:
+  - npm run coveralls
+  # - ./cc-test-reporter format-coverage -p coverage/lcov.info
```
Same here as above. Thanks

kenzdozz replied:
Comments now removed.

kenzdozz commented:
I was trying out codeclimate test coverage. I will remove the comments. Thanks

kenzdozz committed:
chore(setup-codeclimate): Integrating codeclimate maintainability and test coverage
- Integrating codeclimate maintainability and test coverage and add badges to ReadMe.
[Delivers #163496103]

kenzdozz commented:
> Hi @kenzdozz, any reason why you have 10 commits for this task. I think you should combine them into one commit. Thanks

I have combined the commits to one

ascii-dev reviewed (approved):
LGTM
